packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.styles.ts

```diff
@@ -49,10 +49,9 @@ export const getMultiStackedBarChartStyles = (props: IMultiStackedBarChartStyleP
     opacityChangeOnHover: {
       opacity: shouldHighlight ? '' : '0.1',
       cursor: href ? 'pointer' : 'default',
-      stroke: theme.palette.white,
-      strokeWidth: 2,
       selectors: {
         '&:focus': {
+          outlineColor: theme.palette.black,
           stroke: theme.palette.black,
```

Comment:
is outlineColor and stroke different

Reply:
![image](https://github.com/microsoft/fluentui/assets/70366079/c118246f-8b4e-49df-a2d1-2d1c24afe771)

In Dark Mode, focusZone outlineColor doesn't change automatically.